### DISCUSSION TITLE
原神兑换码

### DISCUSSION THREAD

**Discussion by MrDucker** [share121/inter-knot#2417] [其他]
5.0版本「荣花与炎日之途」前瞻特别节目兑换码（国服）：

XX3PD82ZYDGN

9FJ7U83HYDJA

5WJNV92GGUJW 
任务提前开启
圣遗物自选词条 (1主2副)，道具在分解5星圣遗物，周年庆邮件，纪行和大世界都会给
地灵龛原石40→80
5.0 5.1 5.3完成主线给500*3原石
星之归还特定条件触发+10抽
常驻自选五星 (每年周年庆给)
七天神像任意地区均可提升体力上限
武器池定轨值上限从2→1（期望抽数少20抽）
小保底不歪概率提升5%（期望抽数少3抽）
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/e03be25a7b2bc86f2b1909dddb6ca915_6686505940047795068.png" alt="安东-元气满满" width="75" >

**Comment by did1181** [comment]
朋友，而你才是真正的英雄😃
